docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -1,15 +1,158 @@
 ---
 title: Create an event
 description:
-  This guide explains how to create an event in Front-Commerce and dispatch it to external services.
+  In this guide, we will cover the creation of an emitter and how to implement
+  them in your pages, we will create an Emitter that Emit an event when a
+  customer reach a page called FAQ.
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## What is an event emitter
 
-- Create an event emitter
-- Emit an event from your code
+The events are dispatched through the use of an event emitter. The role of the
+event emitter is to push events to the server event pipeline to be processed by
+the server.
+
+Emitter are implemented in Domain Event class that define an event related to a
+specific event for your app.
```

Comment:
**issue:** I don't this this sentence makes sense, emitters aren't domain event classes, server events are 😕. I think the domain event class you're referring to are `ServerEvent`s. Server events are "emitted" through an `EventEmitter` (which is a [standard node feature](https://nodejs.org/api/events.html#class-eventemitter)) to be pushed on the server event pipeline, and then be processed.

Reply:
i'll remove it then

Reply:
Fixed with your suggestion